src/components/Header/Header.js

```diff
@@ -48,17 +49,17 @@ class Header extends Component {
               </Link>
             </div>
           </div>
+
           <div className="small-screen-3 medium-screen-2 large-screen-2 right-align">
             <Button buttonClass="button header-search-button white">
               <FontAwesomeIcon icon={faSearch} size="lg" />
             </Button>
-            <Button buttonClass="button header-notification-button white">
-              <FontAwesomeIcon icon={faBell} size="lg" /> <span className="number">{10}</span>
-            </Button>
+            <Notification className="right " />
             <span className="wrap-header-user-button">
               <Button buttonClass="button header-user-button white" onClick={this.toggleUserMenu}>
                 <HeaderUserImage />
               </Button>
+
```

Comment:
I tested your PR in my local machine.
Search, notification button and profile avatar are not aligned. Fix them

Reply:
Done